spot-client/src/spot-remote/ui/views/join-code-entry.js

```diff
@@ -78,7 +76,6 @@ export class JoinCodeEntry extends React.Component {
      */
     componentDidMount() {
         this.props.updateReadyStatus(true);
-        this.props.ultrasoundService.setMessage('');
```

Comment:
is that not needed anymore, because ultrasound listens for Redux state and it will track the join code changes?

Reply:
In current master, withUltrasound already sets the message to undefined on unmount. To when the remote control view unmounts it should already be clearing the message. The way I'm interpreting the code right now is this call to set an empty string is redundant.